msteams-platform/concepts/build-and-test/apps-package.md

```diff
@@ -13,6 +13,7 @@ An app package is a file format that has the required resources to install and r
 
 * **[App manifest](#app-manifest)**: Describes how your app is configured, including its capabilities, required resources, and other important attributes.
 * **[App icons](#app-icons)**: Each package requires a color and outline icon for your app.
+* **[Custom activity icons](/graph/teams-send-activityfeednotifications?toc=%2Fmicrosoftteams%2Fplatform%2Ftoc.json&bc=%2Fmicrosoftteams%2Fplatform%2Fbreadcrumb%2Ftoc.json&tabs=desktop%2Chttp#custom-activity-icons-in-activity-feed-notifications)**: Tailor-made icons that you can use in activity feed notifications.
```

Comment:
Can't we redirect to specific section which actually explains how to configure it and what configuration we need to add it in the manifest?

Reply:
ok fine.